affiliation/service.go

```diff
@@ -61,6 +61,7 @@ type Service interface {
 	DeleteOrgDomain(context.Context, *affiliation.DeleteOrgDomainParams) (*models.TextStatusOutput, error)
 	GetListProfiles(context.Context, *affiliation.GetListProfilesParams) (*models.GetListProfilesOutput, error)
 	GetIdentity(context.Context, *affiliation.GetIdentityParams) (*models.IdentityDataOutput, error)
+	GetIdentityByUser(ctx context.Context, params *affiliation.GetIdentityByUserParams) (*models.IdentityDataOutput, error)
```

Comment:
Remove that parameter name `params` from `params *affiliation.GetIdentityByUserParams` - other signatures are not specifying params names but only types.